cloudify/tests/test_local_workflows.py

```diff
@@ -615,10 +614,8 @@ def blueprint_model(ctx, **_):
             self.assertEqual(['type'], node1.type_hierarchy)
             self.assertEqual(['type', 'cloudify.nodes.Compute'],
                              node2.type_hierarchy)
-            self.assertThat(node1.properties.items(),
-                            ContainsAll({'property': 'value'}.items()))
-            self.assertThat(node2.properties.items(),
-                            ContainsAll({'property': 'default'}.items()))
+            self.assertEqual(node1.properties.get('property'), 'value')
+            self.assertEqual(node2.properties.get('property'), 'default')
```

Comment:
imagine doing 2 calls to `.items()` and a testtools `ContainsAll` to check one single damn value